City-Bureau/city-scrapers-san-diego
# 🏗️ Build spider: San Diego City Council

## What's this PR do?
Adds a spider to scrape meetings from the website of San Diego City Council. The new spider is called `sandie_city_council`.

## Why are we doing this?
Requested by City Bureau's partners in San Diego.

## Steps to manually test

1. Ensure the project is installed:
```
pipenv sync --dev
```
2. Activate the virtual env and enter the pipenv shell:
```
pipenv shell
```
3. Run the spider:
```
scrapy crawl sandie_city_council -O test_output.csv
```
4. Monitor the output and ensure no errors are raised.
5. Inspect `test_output.csv` to ensure the data looks valid. Target website is [here](https://www.sandiego.gov/city-clerk/officialdocs/meetings-calendar?calendar_timestamp=1706774400).

## Are there any smells or added technical debt to note? 
- I've ran `scrapy validate sandie_city_council` and I'm getting 100% validation for all fields.
